{
	"name": "CVE-2016-2048",
	"threat_severity": "Low",
	"public_date": "2016-02-01T00:00:00Z",
	"bugzilla": {
		"description": "user with \"change\" but not \"add\" permission can create objects for ModelAdmin",
		"id": "1302699",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1302699"
	},
	"cvss": {
		"cvss_base_score": "3.5",
		"cvss_scoring_vector": "AV:N/AC:M/Au:S/C:N/I:P/A:N",
		"status": "draft"
	},
	"details": [
		"Django 1.9.x before 1.9.2, when ModelAdmin.save_as is set to True, allows remote authenticated users to bypass intended access restrictions and create ModelAdmin objects via the \"Save as New\" option when editing objects and leveraging the \"change\" permission."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2016-2048\nhttps://nvd.nist.gov/vuln/detail/CVE-2016-2048\nhttps://www.djangoproject.com/weblog/2016/feb/01/releases-192-and-189/"
	],
	"acknowledgement": "Red Hat would like to thank upstream Django project for reporting this issue.",
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:ceph_storage:1.2",
			"fix_state": "Not affected",
			"package_name": "Django",
			"product_name": "Red Hat Ceph Storage 1.2"
		},
		{
			"cpe": "cpe:/a:redhat:ceph_storage:1.3",
			"fix_state": "Not affected",
			"package_name": "Django",
			"product_name": "Red Hat Ceph Storage 1.3"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:5::el6",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat Enterprise Linux OpenStack Platform 5 (Icehouse)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:6",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat Enterprise Linux OpenStack Platform 6 (Juno)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:7",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat Enterprise Linux OpenStack Platform 7 (Kilo)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack-optools:7",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat Enterprise Linux OpenStack Platform 7 (Kilo) Operational Tools"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:8",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat OpenStack Platform 8 (Liberty)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack-optools:8",
			"fix_state": "Not affected",
			"package_name": "python-django",
			"product_name": "Red Hat OpenStack Platform 8 (Liberty) Operational Tools"
		},
		{
			"cpe": "cpe:/a:rhel_sam:1",
			"fix_state": "Not affected",
			"package_name": "Django",
			"product_name": "Red Hat Subscription Asset Manager"
		}
	],
	"upstream_fix": "Django 1.9.2",
	"csaw": false
}
